test/unit/config/global-mocks.js

```diff
@@ -49,3 +50,6 @@ if ( ! global.TextEncoder ) {
 // Override jsdom built-ins with native node implementation.
 global.Blob = BlobPolyfill;
 global.File = FilePolyfill;
+
+// Polyfill structuredClone for jsdom.
+global.structuredClone = structuredClone;
```

Comment:
@tyxla writes:
> So you can't just use the underlying Node.js modules like this (please correct me if I'm missing something).

The `jsdom` environment creates a new global object for the environment, but it still runs in Node.js. If native `structuredClone` is available in Node.js, then you should be able to "forward" it from the Node.js global to the `jsdom` global.

Could the `global.structuredClone = structuredClone` assignment work without importing the CoreJS polyfill? If the `structuredClone` reference uses the Node.js `globalThis`, it should work.

Reply:
I tried that initially, but unfortunately, it doesn't work.

Reply:
OK then let's do the CoreJS version.